byway/pom.xml

```diff
@@ -24,7 +24,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
-   
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>  
+
```

Comment:
this adds a duplicate copy of `appengine`. Please check this file back out to the version in master.

Reply:
Done.